AEPIdentity/Sources/Identity+PublicAPI.swift

```diff
@@ -143,3 +156,17 @@ import Foundation
         }
     }
 }
+
+extension CustomIdentity {
+    internal static func from(dict: [String: Any]) -> CustomIdentity? {
+        guard let type = dict[CodingKeys.type.rawValue] as? String,
+              let origin = dict[CodingKeys.origin.rawValue] as? String,
+              let identifier = dict[CodingKeys.identifier.rawValue] as? String,
+              let rawState = dict[CodingKeys.authenticationState.rawValue] as? Int,
+              !identifier.isEmpty else {
+            return nil
```

Comment:
I assume these ids should be already cleanup up by the internal Identity extension, but in case this fails the conversion let's log a debug message

Reply:
Another option is to not have this check as the values are coming directly from the extension, and simply return the values given.